package.json

```diff
@@ -57,7 +57,7 @@
     "valid-url": "=1.0.9",
     "validator": "=7.0.0",
     "validator.js": "=2.0.3",
-    "wamp-socket-cluster": "LiskHQ/wamp-socket-cluster#47b53a59f72441686314296f619b67cc8e855e21",
+    "wamp-socket-cluster": "LiskHQ/wamp-socket-cluster#development",
```

Comment:
but i think that you are right and a safer option would be to update the wamp-socket-cluster version manually each time